src/test/java/hudson/views/RegExJobFilterTest.java

```diff
@@ -301,44 +300,6 @@ public void testMaven() {
 			assertFalse(mavenRegex("bar").matches(jobOfType(type).mavenBuilder("", "", "", "Foobar").asItem()));
 			assertTrue(mavenRegex(".*bar").matches(jobOfType(type).mavenBuilder("", "", "", "Foobar").asItem()));
 		}
-
-		for (JobType<? extends Job> type: availableJobTypes(MAVEN_MODULE_SET)) {
-			for (JobMocker.MavenBuildStep step : JobMocker.MavenBuildStep.values()) {
```

Comment:
seemed to be related to the suspended `m2-extra-steps-plugin`, I wasn't able to see how to adapt / if it's still necessary.